_data/identity.yml

```diff
@@ -1,4 +1,4 @@
-websites:
+﻿websites:
```

Comment:
What is going on here? Why is this showing up as changed? 🤔

Reply:
I did a diff locally and it didn't show any of those changes. Must be a glitch with GitHub.

Reply:
have no idea why it happened. I didn't do it ;)

Reply:
I looked at this file using mcedit and it shows that the first char is unicode `FEFF`. It seems like some text editors add it..? @w3ua could you please try to remove it?